tests/unit/test_amazon_estimator.py

```diff
@@ -484,7 +529,7 @@ def test_is_latest_xgboost_version():
             assert _is_latest_xgboost_version(version) is False
 
     assert _is_latest_xgboost_version("0.90-1-cpu-py3") is False
-
+    assert _is_latest_xgboost_version("0.90-2-cpu-py3") is False
```

Comment:
Can we just perform a loop over the supported versions, excluding the last entry? That way we don't need to add a new line each time there's a new version.